packages/react-core/package.json

```diff
@@ -28,7 +30,7 @@
     "rollup-pluginutils": "^2.8.2"
   },
   "dependencies": {
-    "@scalprum/core": "file:../core",
+    "@scalprum/core": "0.0.0",
```

Comment:
I think this should be `*` so it always pulls the linked dependency no matter what.